app/src/main/java/ch/epfl/cs311/wanderwave/ui/screens/TrackListScreen.kt

```diff
@@ -75,6 +75,8 @@ fun TabContent1(
   val uiState by viewModel.uiState.collectAsStateWithLifecycle()
   var searchQuery by remember { mutableStateOf("") }
 
+  LaunchedEffect(Unit) { viewModel.updateBannedSongs() }
```

Comment:
For this, I have one concerned. So now i can't really test it, because it's not fully implementented(the banned songs), but if im on the track list screen, and banned a song, will the effect happens. 
(Ex: there is a song in my recently song and liked song, i ban it from my liked songs, will it appear in the recently songs. imo no, bc we have to fetch the profile again. But this is just a guess, as i can't test it, maybe it works)